annotations/anchor.md

```diff
@@ -0,0 +1,21 @@
+**_anchor_** (English); _anker_ (Danish); _Anker_ (German)
+
+_**akkeri** n., pl. akkerum_ (Old Norse) [citations: [prose](https://onp.ku.dk/onp/onp.php?o1281)/[poetry](https://lexiconpoeticum.org/m.php?p=lemma&i=2907)]  
+
+An object designed to prevent or slow the drift of a ship, by gripping the solid seabed under the body of water.  
+
+<div align="center">
+  
+  ![anchor from Ladby ship](../images/Anchor_Ladby.png)  
```

Comment:
We don't have permission to use this image. I need to take them out of the repo.

We should maybe use [the one from Oseberg here](https://www.nb.no/items/URN:NBN:no-nb_digibok_2019111248002?page=335) and maybe also the separate image of the stock from Nicolaysen Plate IV Figure 17